src/rssitem.cpp

```diff
@@ -260,4 +262,26 @@ void RssItem::set_feedptr(const std::weak_ptr<RssFeed>& ptr)
 	feedptr_ = ptr;
 }
 
+void RssItem::download_full_page(ConfigContainer& cfg)
+{
+	CurlHandle handle;
+	const std::string content = utils::retrieve_url(this->link(), handle, cfg, "", nullptr,
```

Comment:
`retrieve_utl` returns an empty string if it fails to fetch anything. I think we should propagate the error here by returning a `bool` indicating success.

If you're in the mood for some refactoring, you can actually change `retrieve_url` to return a struct describing the error (if any), return it from here, and use it to display a meaningful error message in the item view.